docs/dev/core/internal/upload.md

```diff
@@ -0,0 +1,69 @@
+# Upload mechanism
+
+The `glean-core` Rust crate does not handle the ping upload directly.
+Network stacks vastly differ between platforms, applications and operating systems.
+It is best to use the available platform capabilities to implement any network communication.
```

Comment:
```suggestion
The Glean SDK leverages the available platform capabilities to implement any network communication.
```

? I merely wanted to remove the "it is best" part :-) No strong opinion on this, if you feel strongly about keeping it!